addons/base_vat/views/res_partner_views.xml

```diff
@@ -6,9 +6,13 @@
             <field name="name">view.partner.base.vat.form</field>
             <field name="inherit_id" ref="base.view_partner_form" />
             <field name="arch" type="xml">
-                <xpath expr="//div[hasclass('alert')]" position="after">
-                    <div class="alert alert-warning oe_edit_only" role="alert" attrs="{'invisible': [('vies_failed_message', '=', False)]}">
-                        <field name="vies_failed_message" nolabel="1" readonly="1"/>
+                <xpath expr="//field[@name='vat']" position="replace">
```

Comment:
Hello @dbkosky ,

This `xpath` breaks the Partner Autocomplete functionality for VAT numbers.

In the `partner_autocomplete` module, we have an `xpath` that adds the `field_partner_autocomplete` widget on the VAT field, see [here](https://github.com/odoo/odoo/blob/saas-16.3/addons/partner_autocomplete/views/res_partner_views.xml#L14).

Since your `xpath` replaces the `vat` field, the `field_partner_autocomplete` widget isn't present anymore when this module is installed.

Reply:
@lobaudoux Ah, I'm sorry! :grimacing:  
I've had a go at redefining this xpath in such a way as not to replace the node, but a solution is not forthcoming. Do you have any ideas?

Reply:
@lobaudoux , ok @aliyatastemirova has come up with a solution.
It's not pretty, but it works.
https://github.com/odoo/odoo/pull/138528